include/boost/geometry/algorithms/discrete_hausdorff_distance.hpp

```diff
@@ -39,6 +28,11 @@
 #include <boost/geometry/strategies/distance_result.hpp>
 #include <boost/geometry/util/range.hpp>
 
+#include <algorithm>
+#include <iterator>
+#include <utility>
+#include <limits>
```

Comment:
I recall that this order of headers is prefered in general. In our case this would be Boost.Geometry headers, then other Boost headers, then standard headers. Do we want to make a guideline about this?

I'd add one thing. The headers are not sorted alphabetically. They probably should.

Reply:
I see that in other file the order is different so I'm curious why were they moved here?